libs/api/domains/auth/src/lib/models/consentScopeNode.model.ts

```diff
@@ -0,0 +1,23 @@
+import { Field, ID, ObjectType } from '@nestjs/graphql'
+
+@ObjectType('AuthConsentScopeNode')
+export class ConsentScopeNode {
```

Comment:
If can reuse the `ApiScopeTree` in the rest backend we should be able to reuse the https://github.com/island-is/island.is/blob/main/libs/api/domains/auth/src/lib/models/scopeTreeNode.model.ts#L5 type here.

Reply:
`ApiScopeTree` doesn't have a `hasConsent` flag..

Reply:
I see. Makes sense that this is a new schema in GQL.